bootstrap.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+VENV_DIR="venv"
+if [ ! -d "$VENV_DIR" ]; then
+    virtualenv --python=python3 venv
+fi
+
+# Activate venv and install requirements
+source venv/bin/activate
+pip install -r requirements.txt
+
+# create .env vars and run setup.
+cp settings/.env.example settings/.env
```

Comment:
Don't do this by the user. You should include this step in the readme, because the user needs to set.

Reply:
Given the changes proposed by @ocampor I think we can delete the file **bootstrap.sh** all along. Do you agree @joseluisvaz ?

Reply:
Si